packages/salesforcedx-utils/test/jest/cli/commandBuilder.test.ts

```diff
@@ -0,0 +1,83 @@
+import { Command } from '../../../src/cli/command';
+import {
+  CommandBuilder,
+  FATAL,
+  JSON_FLAG,
+  LOG_LEVEL_FLAG
+} from '../../../src/cli/commandBuilder';
+
+describe('CommandBuild unit tests', () => {
+  const testCommandStr = 'Test Command';
+  const testDescription = 'This is a test description';
+
+  let commandBuilderInst: CommandBuilder;
+
+  beforeEach(() => {
+    commandBuilderInst = new CommandBuilder(testCommandStr);
+  });
+
+  it('Should be a be able to create an instance and build a command.', () => {
+    expect(commandBuilderInst).toBeDefined();
+    expect(commandBuilderInst).toBeInstanceOf(CommandBuilder);
+    const testCommand = commandBuilderInst.build();
+    expect(testCommand).toBeInstanceOf(Command);
+    expect(testCommand.command).toBe(testCommandStr);
```

Comment:
Note: I sort of figured out as I went that `.toBe` and `.toEqual` are pretty much the same but toBe won't work sometimes which led me to search for the diff where I found https://dev.to/thejaredwilcurt/why-you-should-never-use-tobe-in-jest-48ca#:~:text=toEqual%20and%20the%20.,equality%20fails%20on%20non%2Dprimatives.

So bottom line we should probably just default to `toEqual`
I'll update these tests to get rid of `toBe`